semantic/src/site/globals/site.overrides

```diff
@@ -11,8 +15,112 @@ p strong {
   font-weight: @semiboldWeight;
 }
 
-.ui.basebody.container {
-  background: rgb(248, 248, 248);
+#main-container {
+  padding: 0;
+  position: relative;
+  min-height: 100vh;
+  // Max width is 1027px, from Semantic-UI Container
+}
+
+#user-area {
+  font-family: @userAreaFont;
+  color: @userAreaText;
+  display: flex;
+  position: fixed; // TO-DO: No fixed on mobile
+  background-color: @headersInteractiveLow;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 20px;
+  padding-right: 20px;
+  height: @headerHeight; // TO-DO: Adjust for mobile
+  justify-content: space-between;
+  //   top: 0,
+  //   zIndex: 1000,
+}
+
+#user-area.ui.container {
+  margin-left: 0px !important;
+  margin-right: 0px !important;
+}
+
+#user-area-left {
+  display: flex;
+  flex-direction: column;
+}
+
+#menu-bar {
+  // border: 1px white solid;
+}
+// Menu bar links
+#user-area a {
+  color: @userAreaText;
+  font-size: 14px;
+}
+
+// Not currently used, but should be appplied to the Link in the menu
+// bar that reflects the current page
+.selected-link {
+  font-weight: bold;
+}
+
+#org-selector {
+  font-size: 27px;
+  letter-spacing: 3.4px;
+  text-transform: uppercase;
+  // border: 1px white solid;
+  display: flex;
+  align-items: center;
+  height: 100%;
+  max-height: 65px;
+  margin-top: 10px;
+}
```

Comment:
I noticed a slight difference between what is in Invision and here - not sure exactly what is the difference, maybe you can tell, it can be unimportant but I just wanted to flag that here:
![Screen Shot 2021-04-20 at 10 13 11 AM](https://user-images.githubusercontent.com/16461988/115309918-0df74100-a1c1-11eb-8466-0c50f1cc9923.png)
<img width="379" alt="Screen Shot 2021-04-20 at 10 13 00 AM" src="https://user-images.githubusercontent.com/16461988/115309962-1bacc680-a1c1-11eb-89b0-acce319ab40c.png">

Reply:
Okay, I have made the sizes a little closer. The other main difference is the font -- we're using Arial and hers is SF Pro. Still waiting on a suitable replacement for that.

Reply:
I'm trying "Roboto" now -- was an online recommendation for alt to SF. Seems okay, will see what Deb comes back with

Reply:
Thanks for researching on that. It should be a good alternative, and let's see what she thinks. Indeed.
I'm still thinking the company title loos different then what was designed, not sure why. But it seems to big:
<img width="1130" alt="Screen Shot 2021-04-21 at 9 42 32 AM" src="https://user-images.githubusercontent.com/16461988/115467887-588ec100-a286-11eb-867c-04c914d499ae.png">


Specially with longer titles . It might be a good idea to define the company title area and use some `overflow`?

Reply:
Oh, I was looking to the old version before. I think the size look better on the latest. Nevermind : )

<img width="726" alt="Screen Shot 2021-04-21 at 9 50 03 AM" src="https://user-images.githubusercontent.com/16461988/115468279-f6828b80-a286-11eb-96a1-2e9defacdc0e.png">